src/main/java/org/opensearch/ad/util/DateUtils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.ad.util;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoField;
+
+public class DateUtils {
+    public static final ZoneId UTC = ZoneId.of("Z");
+
+    /**
+     * Get hour of day of the input time in UTC
+     * @param instant input time
+     *
+     * @return Hour of day
+     */
+    public static int getHourOfDay(Instant instant) {
```

Comment:
can this be renamed to make it more explicit that it's getting the UTC hour of day?

Reply:
yes